src/utils.ts

```diff
@@ -91,10 +91,444 @@ export async function checkTraitsUrl(traitsUrl: string) {
   }
 }
 
-export async function checkUrlIsUp(url: string) {
+export function validateSpecificParamsViewData(params: {
+  signedMessage?: string | undefined;
+  signableMessage?: any;
+  stream?: boolean | undefined;
+  fwdAllHeaders?: boolean | undefined;
+  fwdHeaderKeys?: string | undefined;
+  mvxNativeAuthEnable?: number | undefined;
+  mvxNativeAuthAccessToken?: string | undefined;
+  mvxNativeAuthMaxExpirySeconds?: number | undefined;
+  mvxNativeAuthOrigins?: string[] | undefined;
+  fwdHeaderMapLookup?: any;
+  _fwdHeaderMapLookupMustContainBearerAuthHeader?: boolean | undefined;
+  _mandatoryParamsList: string[]; // a pure JS fallback way to validate mandatory params, as typescript rules for mandatory can be bypassed by client app
```

Comment:
I would exclude the `_mandatoryParamsList` from params and add it as a separate param to use the advantage of object assigning directly

i.e: 
```typescript
let params  = {signedMessage?: string | undefined,
  signableMessage?: any,
  stream?: boolean | undefined,
  fwdAllHeaders?: boolean | undefined,
  fwdHeaderKeys?: string | undefined,
  mvxNativeAuthEnable?: number | undefined,
  mvxNativeAuthAccessToken?: string | undefined,
  mvxNativeAuthMaxExpirySeconds?: number | undefined,
  mvxNativeAuthOrigins?: string[] | undefined,
  fwdHeaderMapLookup?: any,
  _fwdHeaderMapLookupMustContainBearerAuthHeader?: boolean | undefined,
  };
validateSpecificParamsViewData(params, ['signedMessage', ... ]);
```

Reply:
as above